recipes/termcap/all/conanfile.py

```diff
@@ -85,6 +93,7 @@ def _termcap_path(self):
         return os.path.join(self.package_folder, "bin", "etc", "termcap")
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_target_name", "Termcap")
         self.cpp_info.names["cmake_find_package"] = "Termcap"
         self.cpp_info.names["cmake_find_package_multi"] = "Termcap"
```

Comment:
Where did it come from?

Reply:
True, termcap doesn't offer official targets, and I didn't find any official cmake module on cmake docs. I'll remove all.

Reply:
Done, removed.